.github/hacktoberfest.md

```diff
@@ -1,40 +1,53 @@
 
-To have your review of a translation PR recognized for Hacktoberfest, open a PR to this file adding to the end of the reviewer log below. We’ll add the `hacktoberfest-accepted` label to make sure it’s recognized.
+### Themes
 
-## i18n PR reviewer log
+#### Stars Around the Globe
 
-These awesome people reviewed i18n PRs for us. Thank you! 💜
+Astro's official docs framework Starlight welcomes UI translations for any language! 
 
-- @agustinmulet reviewed #1609
-- @moustaphadev reviewed #1668
-- @moustaphadev reviewed #1654
-- @AbrahamX3 reviewed #1713
-- @AbrahamX3 reviewed #1747
-- @AbrahamX3 reviewed #1711
-- @AbrahamX3 reviewed #1712
-- @AbrahamX3 reviewed #1706
-- @AbrahamX3 reviewed #1710
-- @AbrahamX3 reviewed #1705
-- @breadadams reviewed #1767
-- @breadadams reviewed #1930
-- @moustaphadev reviewed #1656
-- @moustaphadev reviewed #1890
-- @moustaphadev reviewed #1900
-- @moustaphadev reviewed #1902
-- @moustaphadev reviewed #1909
-- @moustaphadev reviewed #1917
+Learn more on how to contribute with a UI translation in [Starlight's contribution guide](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md#translating-starlights-ui).
+
+#### World Tour Comeback 
+
+A few languages of Astro Docs' are behind its colleagues - What are you waiting for?! Help Arabic, Korean, Traditional Chinese, and Polski docs come back into the fight!
+
+Learn more on [Astro Docs' translation guide](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md).
+
+#### Polyglot Astro 101 
+
+Astro Docs features its own 32-part beginner tutorial in several languages. Let's help other newcomers learn Astro around the world today! 
+
+Learn more on [Astro Docs' translation guide](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md) and in [its own Tutorials section](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md#tutorials).
+
+#### Here Comes a New Fighter!
+
+Astro Docs just added official support for Hindi translations. Let's show how strong its community is!
+
+Learn more on [Astro Docs' translation guide](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md).
+
+#### Global Level Reviews 
+
+No translation is approved without a careful double-check! Review a page and increase the document's [i18n reviews count](#i18n-reviews-count).
+
+Learn more on [Astro Docs' translation guidance for reviewers](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md#reviewing-someone-elses-pr).
+
+## i18n Reviews Count
+
+Reviewed a PR? Create a PR increasing the count below to have your contribution considered!
+
+During Hacktoberfest 2023, we've received 0 PR reviews from helpful contributors. Thank you!
```

Comment:
I know there was some discussion around whether the list format we used last year was hard to manage. FWIW this format _guarantees_ a merge conflict between updates. That’s OK if we prefer handling that manually each time and like this format, but flagging it just so we’re all aware.

Reply:
I'm pretty sure the list format, in practice, guaranteed a merge conflict every time.  :sweat_smile: 

We have to stop and confirm the contribution by looking up the PR anyway, so updating a number doesn't feel nearly as much work as the list from last time. (Still remember the same person who added themselves to "the end of the same list" in the same spot 3 times for 3 PRs... Still open for suggestions though, especially for whatever data we want to capture!

e.g. Did we want to do these per-language, too? Have a count for each language?

Reply:
> e.g. Did we want to do these per-language, too? Have a count for each language?

I feel like just having the count is enough.